### ISSUE TITLE
single logout on multidomain

### ISSUE THREAD

**Issue by gagalago** [nextauthjs/next-auth#440]
**Your question**
How can we achieve to logout the user on other domains when he log out from one? 

**What are you trying to do**
We are investigating to use this lib to manage the authentication of users and we wanted to know if this lib can fit our requirement:
- using auth0 behind the scene (OK)
- allow silent authentication (see https://github.com/iaincollins/next-auth/issues/439)
- allow single logout on multidomain
you can see a [similar question asked on auth0](https://community.auth0.com/t/single-logout-on-multi-domains-with-refresh-token-rotation/46405)

we see that log out from one tab is propagated to other tabs for the same domain. but how to achieve that when the website uses multiple domains?

**Documentation feedback**
don't see anything in the documentation about multidomain

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [x] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by iaincollins** [comment]
Thanks for asking about this. I've left an answer which should cover this in #439!

tl;dr it's not supported but happy to roll up both into a feature request
